api/v1beta1/containerjfr_types.go

```diff
@@ -93,6 +93,7 @@ type PersistentVolumeClaimConfig struct {
 // +kubebuilder:resource:path=containerjfrs,scope=Namespaced
 
 // ContainerJFR is the Schema for the containerjfrs API
+//+operator-sdk:csv:customresourcedefinitions:resources={{Deployment,v1},{PersistentVolumeClaim,v1},{Secret,v1},{Service,v1},{Route,v1},{FlightRecorder,v1beta1},{ConsoleLink,v1}}
```

Comment:
Maybe remove FlightRecorder here, but add `{Ingress,v1}`? This is coming with #160